src/creature.cpp

```diff
@@ -814,6 +814,11 @@ void Creature::add_effect( efftype_id eff_id, int dur, body_part bp,
                 e.mod_intensity(e.get_int_add_val());
             }
 
+            // Force intensity if it is duration based
+            if (e.get_int_dur_factor() != 0) {
```

Comment:
Style issue: no space in front of the `(`, but a space after it. And another space before the `)`. See doc/CODE_STYLE.txt

Applies to the other added code as well. (Function declaration and definition are fine as they are now.)

And I agree with questions. Why are you adding and removing empty lines seemingly at random?